spring-boot-project/spring-boot/src/test/java/org/springframework/boot/cloud/CloudPlatformTests.java

```diff
@@ -50,7 +50,17 @@ void getActiveWhenNotInCloudShouldReturnNull() {
 		Environment environment = new MockEnvironment();
 		CloudPlatform platform = CloudPlatform.getActive(environment);
 		assertThat(platform).isNull();
+	}
 
+	@Test
+	void getActiveWhenHasWebsiteSiteNameShouldReturnAzureAppService() {
```

Comment:
It would be good to have a test or two that asserts that `AZURE_APP_SERVICE` isn't active when only one of the environment variables is set.

Reply:
Thank you for your review, I have added 2 tests